plans/variables_plan.fmf

```diff
@@ -2,13 +2,20 @@ summary: Basic variables test
 execute:
     how: tmt
     script: |
+        set +x
+        echo "Set final_result to 0 as initial value"
         final_result=0
         # Test if variable $SOME_DUMMY_VARIABLE is 'FOO_BAR_VARIABLE'
```

Comment:
Just a nitpick, but I think that now are the comments not necessary, as you added the echo line.